src/check_synced.rs

```diff
@@ -0,0 +1,68 @@
+//! Validates that remote data matches data in team repo.
+
+use crate::data::Data;
+use crate::github::{self, GitHubApi};
+use crate::schema;
+use log::error;
+use rayon::prelude::*;
+use std::collections::HashMap;
+
+pub(crate) fn check(data: &Data) -> Result<(), failure::Error> {
+    const BOT_TEAMS: &[&str] = &["bors", "bots", "rfcbot", "highfive"];
+    let github = GitHubApi::new();
+    let mut remote_teams = github
+        .teams()?
+        .into_par_iter()
+        .filter(|t| !BOT_TEAMS.contains(&t.name.as_str()))
+        .map(|team| {
+            let members = github.team_members(&team.name)?;
+            Ok((team.name.clone(), (team, members)))
+        })
+        .collect::<Result<HashMap<_, _>, failure::Error>>()?;
+
+    for team in data.teams() {
+        let local_teams = team.github_teams(&data)?;
+        let local_team = local_teams.into_iter().find(|t| t.org == "rust-lang");
```

Comment:
Hm, I think it would be worthwhile to have a const or similar with a list of orgs we validate like this. For example, rust-lang-ci and rust-lang-nursery likely should also be included here.

Reply:
I agree. I didn't want to complicate things initially with other orgs, but we should definitely do this - though perhaps in a follow-up.